docs/sources/administration/cli.md

```diff
@@ -10,41 +10,213 @@ weight = 8
+```
+
+### Override a configuration setting  
+> In plain terms, what does this mean? The text below was copied from the Grafana CLI help.
+Configuration options to override defaults as a string. For example: cfg:default.paths.log=/dev/null
+
+**Example:**
+```bash
+grafana-cli --configOverrides cfg:default.paths.log=/dev/null plugins install <plugin-id>
+```
+
+### Override homepath value  
+
+Sets the path for the Grafana install/home path, defaults to working directory. You do not need to use this if you are in the Grafana installation directory when using the CLI.
+ 
+**Example:**
+```bash
+grafana-cli admin reset-admin-password --homepath "c:\Program Files\grafana" mynewpassword
+```
+
+### Override config file         
+
+`--config value` overrides the default location where Grafana expects the configuration file. Refer to [Configuration]({{< relref "configuration.md" >}}) for more information about configuring Grafana and default configuration file locations.
+
+**Example:**
+```bash
+grafana-cli admin reset-admin-password --config "/etc/configuration/" mynewpassword
+```
+
+## Plugins commands
+
+Grafana CLI allows you to install, upgrade, and manage your Grafana plugins. For more information about installing plugins, refer to [plugins page]({{< relref "../plugins/installation.md" >}}).
+
+All listed commands apply to the Grafana default repositories and directories. You can override the defaults with [Global Options](#global-options).
+
+### List available plugins
+
+```bash
+grafana-cli plugins list-remote
+```
+
+### Install the latest version of a plugin
+
+```bash
+grafana-cli plugins install <plugin-id>
+```
+
+### Install a specific version of a plugin
+
+```bash
+grafana-cli plugins install <plugin-id> <version>
+```
+
+### List installed plugins
+
+```bash
+grafana-cli plugins ls
+```
+
+### Update all installed plugins
+```bash
+grafana-cli plugins update-all
+```
+
+### Update one plugin
+
+```bash
+grafana-cli plugins update <plugin-id>
+```
+
+### Remove one plugin
+
+```bash
+grafana-cli plugins remove <plugin-id>
+```
+
+## Admin commands
+
+Admin commands are only available in Grafana 4.1 and later.
+
+### Show all admin commands
+
+```bash
+grafana-cli admin
+```
 
 ### Reset admin password
 
-You can reset the password for the admin user using the CLI. The use case for this command is when you have lost the admin password.
+`grafana-cli admin reset-admin-password <new password>` resets the password for the admin user using the CLI. You might need to do this if you lose the admin password.
+
+If there are two flags being used to set the homepath and the config file path, then running the command returns this error:
+
+> Could not find config defaults, make sure homepath command line parameter is set or working directory is homepath 
 
-`grafana-cli admin reset-admin-password ...`
+To correct this, use the `--homepath` global option to specify the Grafana default homepath for this command:
```

Comment:
This seems like an edge case we should fix vs having it in the docs.

Reply:
@daniellee?

Reply:
Sounds like this issue: https://github.com/grafana/grafana/issues/17735

I agree with Dan that this is something we should just fix but I don't want to hold up this PR review as this is unlikely to be fixed for the 6.6 release.

@marefr @bergquist what do you think about adding this to your backlog for the next major release?

Reply:
I support this, as long as we remember to go back and update the docs once it is fixed.